apps/examples/wifi_manager/wm_test/wm_test.c

```diff
@@ -838,6 +886,99 @@ int _wt_parse_power(struct wt_options *opt, int argc, char *argv[])
 	return -1;
 }
 
+int _wt_parse_add_hostname(struct wt_options *opt, int argc, char *argv[])
+{
+ 	if (argc != 5) {
+		return -1;
+	}
+	opt->path = argv[3];
+	opt->hostname = argv[4];
+	return 0;		
+}
+
+int _wt_parse_stress5_client(struct wt_options *opt, int argc, char *argv[])
+{
+	if (argc < 4 || argc > 14) {
```

Comment:
I can't understand condition of argc < 4. because acccording to line from 904 to 906 it requires at least 6 arguments.

Reply:
Changes Done as requested.